src/models/components/prior_encoder.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.distributions import Distribution
+from torch.distributions.normal import Normal
+
+from ..abc.prior import Prior
```

Comment:
クラス名を変更しないのであれば、import文を変更してください。  
```suggestion
from ..abc.prior import Prior as AbstractPrior
```